sdk/appconfiguration/azure-data-appconfiguration/samples/appconfig_basic_operation.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @brief This sample provides the code implementation to use the App Configuration client SDK for
+ * C++ to get one or more configuration settings.
+ */
+
+#include <azure/data/appconfiguration.hpp>
+#include <azure/identity.hpp>
+
+#include <iostream>
+
+using namespace Azure::Data::AppConfiguration;
+using namespace Azure::Identity;
+
+int main()
+{
+  try
+  {
+    std::string url = "https://<your-appconfig-name>.azconfig.io";
+    auto credential = std::make_shared<Azure::Identity::DefaultAzureCredential>();
+
+    // create client
+    ConfigurationClient configurationClient(url, credential);
+
+    Azure::Response<GetKeyValueResult> response = configurationClient.GetKeyValue("key", "accept");
+
+    std::cout << response.Value.Value.Value() << std::endl;
```

Comment:
This is a known and unfortunate side effect of having a model field named `Value`, along with `Response<T>'s ` Value, and `Nullable<T>'s` Value().

We'd have to consider what we do here given Key:Value is a pervasive concept:
https://github.com/Azure/azure-sdk/pull/6869

Reply:
Based on feedback on the samples from @ronniegeraghty, we'll split this up to be clearer for the reader to understand what each piece in the "value chain" is referring to.